Update local.md with extra info for windows users, edit RHF notebook theory.

emielvb opened the pull request.

xdvriend reviewed (changes requested):
I would not change the assignment in the RHF notebook, as this is intentionally worded this way. That way we can see which students grasp the material and find the "better" way of doing the diagonalization of the generalised eigenkproblem, and which don't. 

If you undo these changes, this PR is good to go, as the windows documentation is a welcome addition.

emielvb commented:
@xdvriend I added the necessary theory sections regarding the orthogonalisation scheme and removed any mention of scipy.

I also changed the way the A matrix was constructed (since this was done using psi4 in the computational QC course) and tested the new code by using the newly constructed A matrix to diagonalise S. (Test was not added in final commit to ensure the students find it for themselves)

xdvriend reviewed (approved):
Looks good to me!

Feel free to merge this PR.
